### ISSUE TITLE
Minimum Curio version?

### ISSUE THREAD

**Issue by eserte** [bluefeet/Catalyst-Model-Curio#1]
On some of my smoker systems the test suite fails:

```
Couldn't instantiate component "MyApp::Model::MultiCache", "Can't locate object method "declared_keys" via package "MyApp::Service::MultiCache" at /usr/perl5.26.0p/lib/site_perl/5.26.0/Catalyst/Test.pm line 155.
BEGIN failed--compilation aborted at t/basic.t line 7.
# Seeded srand with seed '20190817' from local date.
t/basic.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

It seems that this happens if Curio is already installed, but too old. Statistical analysis (theta=0 means "bad"):
```
****************************************************************
Regression 'mod:Curio'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	   2.94
[1='eq_0.04']  	     -0.0000	      0.0000	  -1.70
[2='eq_0.05']  	     -0.0000	      0.0000	  -0.64
[3='eq_0.06']  	      1.0000	      0.0000	8122008390940711.00
[4='eq_0.07']  	      1.0000	      0.0000	3948696594925393.50
[5='eq_0.08']  	      1.0000	      0.0000	8436930989177792.00

R^2= 1.000, N= 74, K= 6
****************************************************************
```

**Comment by bluefeet** [comment]
Hey, thanks for the report.  This is fixed in 5ace0f6e7aae66d4b7db39498cda5a7d24a447ae I just haven't done a release yet.

**Comment by bluefeet** [comment]
Fixed in `0.03`.
